draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -132,31 +132,34 @@
       the Metadata about 5G low latency edge services directly attached to the
       egress routers. 5G <xref target="TS.23.501-3GPP"/>is characterized by
       having edge services closer to the Cell Towers reachable by Local Data
-      Networks (LDN). From IP network perspective, the 5G LDN is a limited
+      Networks (LDN). From an IP network perspective, the 5G LDN is a limited
       domain <xref target="RFC8799"/> with edge services a few hops away from
       the ingress nodes. Only selective UE services are considered as 5G low
       latency edge services.</t>
 
       <t>Note: The proposed edge service Metadata Path Attribute are not
-      intended for the best-effort services reachable via the public internet.
+      intended for the best-effort services reachable via the public Internet.
```

Comment:
agree

Reply:
Note, you didn't have to "agree" to each change.  That just means needing to "resolve conversation" for each of them.  Comment on the things that need to change to reach conclusion.